typescript/packages/well-completions-plot/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { WellCompletionsPlot } from "./WellCompletionsPlot";
+export { CompletionPlotData, PlotData, WellPlotData } from "./types/dataTypes";
```

Comment:
You could include e.g. `Zone` here as well since it's consumed by the `dash/WellCompletions` component.